### DISCUSSION TITLE
META File not loaded

### DISCUSSION THREAD

**Discussion by Arko7777** [mozilla/hubs#3042] [General Questions & Discussion] [answered]
Hi All, 

maybe some of you can help. I searched already a lot that's why I try my best here.

Our hubs stops here and is never ever connected again. Where is the issue? What to change? Anything inside the CORS? 
![image](https://user-images.githubusercontent.com/26479964/93499672-96ec8800-f913-11ea-8a4e-38b413de9aa8.png)

-) and here also the "network" log i got out of it. It seems that the Meta file is not loaded.
![image](https://user-images.githubusercontent.com/26479964/93499702-a075f000-f913-11ea-804f-ec388e39cd5e.png)

Thanks in advance!

**Comment by robinkwilson** [comment, marked as answer]
Quick questions:
1. Do you have a custom client?
1. Did you update any Server Settings in the Admin Panel?

In the AWS Console > EC2 (double check your region!), can you "Terminate" your instance to restart the servers?

Thanks!

**Reply by Arko7777** [reply]
Hi, 
thanks for your feedback first!! :) 

1) No custom clients
2) No server settings from admin panel

I have terminated the EC2 instance. I will check the Meta File again in 2 days. 

Is there anything else what could cause the issue? 
Thanks!

**Reply by Arko7777** [reply]
In the meantime still getting this wss//:80 and https://80 issues. 

My researched showed, that https is not able to run on port:80. May that be the "security" issue while on some private networks the meta data is not loaded? 
![image](https://user-images.githubusercontent.com/26479964/93664341-69682180-fa6e-11ea-9f61-04bc6a0b4cf6.png)

**Reply by robinkwilson** [reply]
You're correct (however, we do encrypt the traffic usign port 80, but networks still don't like it since it's anti-pattern). 

We have some open issues for customizing the ports that are used to bypass the firewall restrictions: https://github.com/mozilla/hubs-cloud/issues/90

We're actively working on this, please stay tuned!

**Reply by Arko7777** [reply]
Thanks @robin-k-wilson 

Can I do it right now manually to make the hubs work?
